qiskit/providers/fake_provider/fake_backend.py

```diff
@@ -119,10 +119,15 @@ def _parse_channels(self, channels):
     def _setup_sim(self):
         if _optionals.HAS_AER:
             from qiskit.providers import aer
+            from qiskit.providers.aer.noise import NoiseModel
 
             self.sim = aer.AerSimulator()
             if self.target and self._props_dict:
-                noise_model = self._get_noise_model_from_backend_v2()
+                with warnings.catch_warnings():
+                    warnings.filterwarnings(
+                        "ignore", category=UserWarning, module="qiskit_aer.noise.device"
+                    )
```

Comment:
Why are we suppressing warnings?  The warnings are something we should look to fix, not hide - either Aer is spuriously warning, or on the Terra side we need to first modify our data so that it doesn't cause warnings; it's the caller here that has the most information to decide how to modify the data to avoid the warnings.  I suspect this is related to Aer issues about spuriously warning, in which case we should not put in the filter, and let Aer fix them.